lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -358,13 +358,10 @@ class MetroItinerary extends NarrativeItinerary {
 
     // Use first leg's agency as a fallback
     return (
-      <div
+      <li
```

Comment:
When viewing one itinerary after a search, the viewed itinerary is still presented as a list with a single item. Instead, there should be no list. Therefore, `<li>` should move out of this component (retain `<div>` for now).